src/layouts/screens/DeleteWarningScreen.jsx

```diff
@@ -34,11 +36,22 @@ export const DeleteWarningScreen = ({ match, onClose }) => {
         })
 
     return fileData ? (
-      <DeleteWarningModal
-        onDelete={() => deleteHandler({ sha: fileData.sha })}
-        onCancel={() => onClose()}
-        type={fileName}
-      />
+      <GenericWarningModal
+        isOpen // Modal is always present for delete screens
```

Comment:
this is not true - if `fileData` is not present, this whole chunk is null; should we instead change it to `isOpen={!!fileData}` or use lodash's `isEmpty` if it's an object

Reply:
We actually do a conditional on `fileData` before deciding to render this modal - I've moved it into the `isOpen` param instead in https://github.com/isomerpages/isomercms-frontend/pull/941/commits/513eedac744b2fe3c2814aff9b935982147c46d2 like the others